src/metabase/driver/postgres.clj

```diff
@@ -56,19 +57,15 @@
 
 (defmethod driver/display-name :postgres [_] "PostgreSQL")
 
-(defmethod driver/database-supports? [:postgres :persist-models]
-  [_driver _feat _db]
-  true)
-
-(defmethod driver/database-supports? [:postgres :persist-models-enabled]
-  [_driver _feat db]
-  (-> db :options :persist-models-enabled))
-
-(doseq [feature [:actions :actions/custom]]
+(doseq [[feature supported?] {:persist-models         (constantly true)
+                              :convert-timezone       (fn [driver _db] (= driver :postgres))
```

Comment:
I know there's prior art checking that the driver is literally `:postgres` but I would have preferred always returning `true` here and then having another `:redshift` method that returns `false`. That way if there's some other `:postgres/with-a-custom-tweak` driver out there somewhere it will still support everything the main `:postgres` driver does unless you specifically disable it.

Reply:
Thanks, I haven't thought of 3rdparty drivers. Will change.